src/screens/ConsentScreen.tsx

```diff
@@ -157,3 +171,20 @@ declare module '@styles' {
 }
 
 export type ConsentScreenStyles = NamedStylesProp<typeof defaultStyles>;
+
+export type CustomConsentScreenProps = {
+  /**
+   * The full Consent and Questionnaire FHIR Resources pertaining to the
+   * consent form. Terms and acceptance text can be derived from here
+   */
+  consentForm: ConsentAndForm | undefined;
+  /** Mutation to accept consent is in flight */
+  isLoadingUpdateConsent: boolean;
+  /** Mutation to accept consent */
+  acceptConsent: () => Promise<void>;
```

Comment:
It would be nice if we used mutate instead of mutateAsync here so that this doesn't return a promise. That way, there's no floating promise to handle, and it's clear that error handling is not required.